SpatialGDK/Documentation/content/toolbars.md

```diff
@@ -17,6 +17,7 @@ Alongside the standard functionality, we’ve added some extra capabilities to t
 
 1. The Unreal<->SpatialOS networking switch
 1. The “Number of Servers” multiplayer option
+1. The "SpatialOS Settings" menu item which opens the [SpatialOS settings](#settings)
```

Comment:
Check the capitalisation on the menu item. Ideally change it to "SpatialOS settings", but if that's non-trivial, leave this doc as it is. If however the menu item is already "SpatialOS settings", update the doc to match :)

Reply:
The menu item in Unreal is with capital S, which is consistent with the Unreal native `Advanced Settings` menu button.

Reply:
Cool, leave as is because it's already consistent.